### ISSUE TITLE
binary frame not supported?

### ISSUE THREAD

**Issue by monkey-jsun** [Pithikos/python-websocket-server#32]
Hi, all,

I'm trying to use this library in my project.  In fact, I'm trying to replace websockets module which uses asyncio.  

However, one immediate issue I'm facing is that it complains about "binary frame not supported".  Is it easy to add binary frame support?

Thanks.

**Comment by Pithikos** [comment]
Yes, it should be relatively easy but unfortunately I don't have the time atm to do any implementation. A workaround is to convert any binary to base64 and send it as text.

**Comment by Ksengine** [comment]
https://github.com/Ksengine/WSocket is a good solution
